88.md

```diff
@@ -0,0 +1,26 @@
+NIP-88
+======
+
+PAY Request
+-----------
+
+`draft` `optional`
+
+At any point, the relay can send a `PAY` request to a client with a BOLT 11 invoice, a description of what the user is purchasing by paying the invoice, and a URL for other options or more information. The Lightning invoice is the ID of the request.
+
+```js
+[
+  "PAY",
+  "<BOLT 11 Invoice>",
+  "<Description>"
+  "<Url for other plan options>"
+]
+```
+
+Supporting clients SHOULD display a popup or notification to describe the action needed and collect immediate response from the user: `pay` and keep using the relay, `dismiss` and stop using the relay, or `more information` by navigating the user to the URL. Clients SHOULD expect that the same request might be sent multiple times, if the user dismisses or pays the amount, the client SHOULD ignore following requests with the same invoice.
```

Comment:
I would leave out the first sentence here. Clients SHOULD handle this in a way that makes sense. A pop-up might not be it.